porcupine/plugins/comment_block.py

```diff
@@ -51,24 +51,15 @@ def comment_or_uncomment(
     return "break"
 
 
-def comment_or_uncomment_in_current_tab() -> None:
-    tab = get_tab_manager().select()
-    assert isinstance(tab, tabs.FileTab)
-    comment_or_uncomment(tab, None)
-
-
-def on_new_tab(tab: tabs.Tab) -> None:
-    if isinstance(tab, tabs.FileTab):
-        tab.textwidget.bind("<Key>", functools.partial(comment_or_uncomment, tab), add=True)
+def on_new_filetab(tab: tabs.FileTab) -> None:
+    tab.bind(
+        "<<FiletabCommand:Edit/Comment Block>>",
+        (lambda event: comment_or_uncomment(tab, None)),
+        add=True,
+    )
+    tab.textwidget.bind("<Key>", functools.partial(comment_or_uncomment, tab), add=True)
```

Comment:
The callbacks are a bit confusing. Maybe `comment_or_uncomment` should take the pressed key (event.char) as argument?